projects/eip-7732-implementation-in-nimbus

```diff
@@ -0,0 +1,65 @@
+# Project Template
+
+`eip-7732: enshrined Proposer Builder Separation` implementation in the Nimbus Consensus Client
+
+## Motivation
+
+Implementing proposer-builder separation (PBS) directly into the Nimbus consensus client helps prevent transaction censorship at the protocol level. This enshrines PBS on the protocol, which currently occurs as a sidecar in the protocol, creating a trustless system between proposers and builders. Additionally, it supports scaling Ethereum by enabling future upgrades like Danksharding and Inclusion lists. Working on this project will further solidify my understanding of the core Ethereum protocol, advancing my knowledge from intermediate to advanced.
```

Comment:
ePBS is not a dependency of danksharding. You can find some motivation to highlight in epf wiki https://epf.wiki/#/wiki/research/PBS/ePBS 
```suggestion
Implementing proposer-builder separation (PBS) directly into the Nimbus consensus client helps prevent transaction censorship at the protocol level. This enshrines PBS on the protocol, which currently occurs as a sidecar in the protocol, creating a trustless system between proposers and builders.  Working on this project will further solidify my understanding of the core Ethereum protocol, advancing my knowledge from intermediate to advanced.
```